org.opent2t.sample.thermostat.superpopular/com.nest.thermostat/js/manifest.xml

```diff
@@ -3,9 +3,9 @@
   <!-- associated voice handler -->
   <voice id="" />
   <!-- associated schema -->
-  <schema id="org.OpenT2T.Thermostat.SuperPopular.Sample" />
+  <schema id="org.opent2t.sample.thermostat.superpopular" />
   <!-- associated onboarding module -->
```

Comment:
This nest thermostat translator has not been updated as part of this PR (none of the other translators have been updated either). So let's ignore whatever that says (other than renaming/casing changes of course).

The Wink Thermostat is under review here. Issue #36 tracks updating all the other translators (Once we agree on this one I can do a series of PRs modifying all the other translators one by one following the same pattern)